tests/jury/conftest.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+
+from jury import Jury
+from jury.metrics import load_metric
+
+_TEST_METRICS = [
+    load_metric("bleu"),
+    load_metric("meteor"),
+    load_metric("rouge"),
+    load_metric("sacrebleu"),
+    load_metric("bertscore", params={"model_type": "albert-base-v1", "device": "cpu"}),
+    load_metric("squad"),
+]
+
+_CONCURRENT_TEST_METRICS = [
+    load_metric("bleu"),
+    load_metric("meteor"),
+    load_metric("rouge"),
+    # load_metric("sacrebleu"),
+    # load_metric("bertscore", params={"model_type": "albert-base-v1", "device": "cpu"}),  # Memory allocation
+    load_metric("squad"),
+]
+
+_STR_TEST_METRICS = ["bleu", "meteor", "rouge", "sacrebleu", "squad"]
+
+
+@pytest.fixture(scope="session")
```

Comment:
My general warning for the all fixtures in this file is being aware of the side effects of making them have large scopes. Particularly, they are fine as long as they are immutable (clearly, some of them aren't) or we are sure (or confident) that the test code and/or source code do not mutate them. Otherwise the tests would be untrustworthy.

Reply:
Noted 👍 I didn't think of that when I wrote it, but currently none of these fixtures (fortunately 😅) whose scope is "session" are mutated during tests. However, making them immutable type would be nice, thanks for the tip 🙂